src/typeDefs/dbSchema.ts

```diff
@@ -1,20 +1,21 @@
+import { PhysicalAddress } from './gqlTypes'
+
 export type Contact = {
   email: string
   phone: string
   website: string
   mapsLink: string
+  address: PhysicalAddress
 }
 
 export type Facility = {
-  id: string
   nameEn: string
   nameJa: string
   contact: Contact
-  healthcareProfessionals: HealthcareProfessional[]
+  healthcareProfessionalIds: string[]
```

Comment:
we likely want ids AND professsionals. When you get a facility, we'll want to have the entities and not just IDs. The alternative would be to use one list and just populate the rest of the data besides the id later, but I think this confuses most people. The other alternative is to make an `Output` entity for each API get request, but I don't think we need to do that now. 

My vote is to have the entity here match what is returned to the user

Reply:
@ermish Hmm I'm pretty sure that at our last dev meeting you said `Facility` should have a list of `HealthcareProfessional` IDs. When we looked through Firestore together I showed you that the HealthcareProfessional was saved to both the `facilities` and `healthcareProfessionals` collections. So just to be clear, you want all of the `HealthcareProfessional` data saved to both collections?

Reply:
no, sorry. 
two arrays

1) `healthcareProfessionalIds` gets saved to the db in the facility. 
2) `healthcareProfessionals` is the array that gets populated when facilities are fetched.  (or coming from the user input)

Reply:
@ermish So, just to make sure we're on the same page, `healthcareProfessionalIds` is already being saved to the db in the facility. The only change you want is to fetch the professionals by ID when a facility is queried, right? You don't want the healthcareProfessional entities to be duplicated across collections, right?

So the schema changes you want is this?

```
type Facility
healthcareProfessionalIds: string[]
healthcareProfessionals: HealthcareProfessional[]
```

Reply:
thats right